dev/release/source_build.sh

```diff
@@ -45,6 +45,8 @@ main() {
     mv "${base_name}/" "${base_name}.tmp/"
     cp -R -d "${base_name}.tmp" "${base_name}"
     rm -rf "${base_name}.tmp/"
+    MESON_SOURCE_ROOT="${base_name}/python" MESON_DIST_ROOT="${base_name}/python" \
+                     python "${base_name}/python/generate_dist.py"
```

Comment:
Perhaps a more descriptive name for `generate_dist.py` would be `replace_subproject_symlinks.py`. Happy to make that change (or any other) here